index.js

```diff
@@ -212,7 +238,8 @@ export default class SlimSlider{
     window.addEventListener('resize', e => {
       clearTimeout(this.resized);
       this.resized = setTimeout(_=> {
-        this.destroy()
+        this.destroy();
+        this.removeAutoPlay();
```

Comment:
`this.removeAutoPlay();` should not be called here, instead move it to the `destroy()` method.